test/dist_autograd_test.py

```diff
@@ -1016,37 +1009,6 @@ def test_backward_autograd_engine_error(self):
                 # Run backwards, and validate we receive an error.
                 dist_autograd.backward([val.sum()])
 
-    @unittest.skipIf(TEST_CONFIG.rpc_backend_name == "PROCESS_GROUP",
-                     "Skipping this test temporarily since ProcessGroupAgent does not report errors on node failures")
-    @dist_init(clean_shutdown=False)
-    def test_backward_node_failure(self):
```

Comment:
Why removing these two tests?

Reply:
The error message in these 2 tests are ThriftClient error messages. I moved these 2 into fb/ folder to make it invisible from OSS.

Reply:
We can definitely move it back to the shared test suite once we go through all failure cases and make sure in those cases, the common OSS layer always throws the same exception type and message for different backends.